pages/dashboards/templates/dashboards/multidisease_serology.html

```diff
@@ -0,0 +1,164 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <a href="#content-start" class="sr-only focus:not-sr-only focus:outline-none focus:ring-2 focus:ring-pp-mid-blue px-3 py-2 bg-white">Skip to content</a>
+  <h1 class="sr-only">Multi-disease serology</h1>
+  <div id="content-start">
+    <div class="prose max-w-none space-y-6 mt-4">
+      {% comment %}
+      <div class="bg-pp-pale-grey/30 border-l-4 border-pp-mid-blue text-pp-dark-blue p-4" role="status" aria-live="polite" aria-atomic="true">
+        <p class="m-0"><strong>All data last updated:</strong> TODO</p>
+      </div>
+      {% endcomment %}
+
+      <h2 id="introduction">Introduction</h2>
+      <p>
+        The COVID-19 pandemic highlighted the importance of serological surveillance in tracking viral
+        transmission dynamics, understanding immune responses, guiding vaccination strategies, and assisting
+        in decisions related to public health. High‑throughput serological assays for SARS‑CoV‑2 were
+        developed very early in the pandemic at KTH and SciLifeLab to enable surveillance of populations
+        globally. For information about work done with SARS‑CoV‑2 during the pandemic, see the
+        <a href="/historical-background/">historical background section</a>.
+      </p>
+      <p>
+        As we move on from the pandemic, it is crucial that serological studies include more pathogens
+        than just SARS‑CoV‑2. One of the capabilities that is part of
+        <a href="/plp-program-background/">SciLifeLab's Pandemic Laboratory Preparedness (PLP) programme</a>,
```

Comment:
I think this page (background text page) will not exist as we decided we will only have one page for PLP (which you are working on). So maybe we should point to it?